key-tester.js

```diff
@@ -115,7 +115,7 @@ function run(form_url, ga_code) {
         });
         $('#ssh-form').submit(function(e) {
             e.preventDefault();
-            var key = $('#form-ssh-key').val();
+            var key = rsa.sanitize($('#form-ssh-key').val());
```

Comment:
Are there occasions where you would not want to sanitize a key? If that's not the case, how about just making it a private function that you call during `rsa.parse`?